src/app.module.ts

```diff
@@ -47,6 +48,7 @@ export const configurationModule = ConfigurationModule.register(configuration);
   imports: [
     // features
     AboutModule,
+    AlertsModule,
```

Comment:
I think this is a good hint that the `AppModule` needs to be a Dynamic Module.

Why: by including a module that includes a controller and respective routes we are exposing this functionality to the end users.

See: https://docs.nestjs.com/fundamentals/dynamic-modules

Reply:
Suggested approach here: https://github.com/safe-global/safe-client-gateway/pull/794

If approved, you can then conditionally enable this via the provided configuration.

Reply:
In anticipation of that, I've removed the `AlertsModule` from the `AppModule` in 2a5abef92a96cd6b43694740b3c3b22ef7d37a50 and disabled the test for now to unblock https://github.com/safe-global/safe-client-gateway/pull/795.